test/unit/writer/settings.test.ts

```diff
@@ -62,9 +62,11 @@ describe('writer/settings', () => {
         );
       });
 
-      // it('rejects if the network is on the wrong realm', async () => {
-      //   return expect(verify(editedInput({ network: '5' }))).rejects.toContain('wrong network');
-      // });
+      it('rejects if using testnet on production', async () => {
+        return expect(verify(editedInput({ network: '5' }))).rejects.toContain(
+          'must be a valid network used by snapshot'
```

Comment:
Need to be changed to `network not allowed` once https://github.com/snapshot-labs/snapshot.js/pull/945 and package updated